src/core/public/chrome/ui/header/header_help_menu.tsx

```diff
@@ -17,14 +17,14 @@
  * under the License.
  */
 
+import * as Rx from 'rxjs';
 import { FormattedMessage } from '@kbn/i18n/react';
 import React, { Component, Fragment } from 'react';
-import * as Rx from 'rxjs';
-
+import { InjectedIntl, injectI18n } from '@kbn/i18n/react';
 import {
   // TODO: add type annotations
   // @ts-ignore
```

Comment:
You should be able to remove most if not all of these `ts-ignore`s as these components have been converted.

Reply:
Thanks! Fixed in https://github.com/elastic/kibana/pull/49797/commits/22d59cb3662184f2acc3dcc314973f7bb04f83a5.